packages/rum/src/domain/record/utils.ts

```diff
@@ -28,3 +28,26 @@ export function assembleIncrementalSnapshot<Data extends BrowserIncrementalData>
     timestamp: timeStampNow(),
   }
 }
+
+export type GroupingCSSRule = CSSGroupingRule | CSSSupportsRule | CSSMediaRule
```

Comment:
At a given point, we do cast a `CSSRule` to `GroupingCSSRule` in order to build the path to that rule. Casting directly to `CSSGroupingRule` would cause an issue in browser that don't support `CSSGroupingRule`

Reply:
Casting is performed by ts compiler not by browsers, so we should not have this kind of issue.

Reply:
My bad, the instruction that was failing is `instanceof`, I mixed two stuff. I'll use `CSSGroupingRule` instead